src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let discount = 0;
```

Comment:
this variable is redundant
You can either just create 4 different variables for `3`, `7`, `20`, `50` or group those variables into one object (advanced variant)